File: test/box2.js
var THREE = require('../build/Debug/threejs');

var box2 = new THREE.Box2(1, 100);
console.log(box2);
console.log(box2.set);
console.log(box2.set(2, 200));
console.log(box2.isEmpty);
console.log(box2.isEmpty());

File: addon/impl/Box2.cpp
#include "Box2.h"

namespace three {

Box2* Box2::Create(long min, long max) {
  return new Box2();
}

Box2* Box2::set(long min, long max) {
  return this;
}

bool Box2::isEmpty() {
  return false;
}

}; // namespace three